### PR TITLE
feat(dependency): remove packages

### PR DESCRIPTION
**Pull request by rudywaltz**

### CODE REVIEW

File: lib/service.ts [hunk 1]
```diff
@@ -57,11 +56,11 @@ export class TranslateService {
...
-    return pipe(
-      split('%s'),
-      map(split('%d')),
-      flattenArray
-    )(text);
+    return text.split('%s')
+      .map(function splitDelimiter(partialText) {
+        return partialText.split('%d')
+      })
+      .flat(Infinity)
```
Comment (peter-vasarhelyi) [inline lib/service.ts]: I think this should work:
`text.split(/%s|%d/gi)`

**Comment by peter-vasarhelyi** [comment]
👍

**Review by boristomic — APPROVED** [review]

File: lib/service.ts (hunk 1 above)
Comment (rudywaltz) [reply lib/service.ts]: done

**Review by peter-vasarhelyi — APPROVED** [review]

**Pull request merged**